security-privacy-questionnaire.md

```diff
@@ -30,50 +30,55 @@ Additionally, user agents could also choose to persist the last directory a file
+No, unless a device exposes such sensors as files or directories. User agents are encouraged to block access to such files or directories (for example `/dev` on linux like systems).
 
-### 2.9. Does this specification enable new script execution/loading mechanisms?
+### 2.9. Do features in this specification enable new script execution/loading mechanisms?
 
 No.
 
-### 2.10. Does this specification allow an origin to access other devices?
+### 2.10. Do features in this specification allow an origin to access other devices?
 
 Not really. The exception would be devices that are exposed as files or directories by the platform. I.e. network shares or cloud storage sync clients could expose data on other devices in a way that looks like regular files or directories. The user agent could let the user pick these files or directories, thereby giving an origin implicit access to this other device. This API doesn't have any functionality to let a website enumerate all network shares on the local network, only explicitly selected files or directories can be accessed by an origin.
 
-### 2.11. Does this specification allow an origin some measure of control over a user agent’s native UI?
+### 2.11. Do features in this specification allow an origin some measure of control over a user agent’s native UI?
 
 The origin can pop up native file or directory pickers, and have some control over what appears inside that native UI (e.g. accepted file types, starting directory and suggested file names), but that control is very limited. The spec does put limitations on what is allowed as an accepted file type and suggested file name to limit the security impact of allowing websites to control the native UI. User agents are expected to employ similar mechanisms to sanitize the sugessted file names as are used to sanitize for example suggested file names in `<a download="foo.ln">` today.
 
-### 2.12. What temporary identifiers might this this specification create or expose to the web?
+### 2.12. What temporary identifiers do the features in this specification create or expose to the web?
 
 The `getUniqueId()` method will create a temporary unique identifier for a given handle. This ID will become invalid if the user clears storage for the site.
 
 ### 2.13. How does this specification distinguish between behavior in first-party and third-party contexts?
 
 It is expected that user agents do not allow third-party contexts to prompt for any kind of access using this API. I.e. third-party contexts can potentially access files or directories that their origin was already granted access to in a first-party context (by sharing handles via IndexedDB or postMessage), but can't trigger any new file/directory pickers or permission requests.
 
-### 2.14. How does this specification work in the context of a user agent’s Private Browsing or "incognito" mode?
+### 2.14. How do the features in this specification work in the context of a browser’s Private Browsing or Incognito mode?
```

Comment:
If we decide to include discussions of the BucketFS in this questionnaire (in a follow-up, presumably), we should mention that it's expected to be implemented using an in-memory file system in this case